boards/nxp/imx95_evk/CMakeLists.txt

```diff
@@ -12,5 +12,49 @@ if (CONFIG_SOF AND CONFIG_BOARD_IMX95_EVK_MIMX9596_M7_DDR)
   )
 endif()
 
+if(CONFIG_BOARD_NXP_SPSDK_IMAGE OR (DEFINED ENV{USE_NXP_SPSDK_IMAGE}
+  AND "$ENV{USE_NXP_SPSDK_IMAGE}" STREQUAL "y"))
+  # The blobs binary is self-extracting file which could be used only in Linux bash
+  if(NOT CMAKE_HOST_UNIX)
+    message(FATAL_ERROR "SPSDK Image generating failed: requires Linux host!")
+  endif()
+  if(CONFIG_BOARD_IMX95_EVK_MIMX9596_M7)
+    set(FIRMWARE_RELEASE "imx95-19x19-lpddr5-evk-boot-firmware-0.1")
+    set(AHAB_CONFIG_FILE "imx95_evk_mimx9596_m7_ahab.yaml")
+  else()
+    message(FATAL_ERROR "SPSDK Image not supported on the platform!")
+  endif()
+  file(WRITE ${CMAKE_BINARY_DIR}/zephyr/${AHAB_CONFIG_FILE}
+    "
+    family: mimx9596
+    revision: a1
+    target_memory: standard
+    output:                   ${CMAKE_BINARY_DIR}/zephyr/flash.bin
```

Comment:
we should use CONFIG_KERNEL_BIN_NAME as in tests/misc/test_build there uses below for output binary

```
    extra_configs:
      - CONFIG_KERNEL_BIN_NAME="A_kconfig_value_with_a_utf8_char_in_it_Bøe_"
```
@yangbolu1991